test/lint/mixed-whitespace.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh -eu
+
+echo "Checking for mixed tabs and spaces in shell scripts..."
+
+OUT=$(git grep -P '\t' . 2>/dev/null | grep '\.sh:' || true)
```

Comment:
If the list of `.sh` files containing tabs in them is desired the following sounds simpler:
`git grep -l -P '\t' '*.sh'`